packages/framework-provider-local/src/services/read-model-registry.ts

```diff
@@ -0,0 +1,40 @@
+import { ReadModelEnvelope, UUID } from '@boostercloud/framework-types'
+import * as DataStore from 'nedb'
+import { readModelsDatabase } from '../paths'
+
+export class ReadModelRegistry {
+  public readonly readModels: DataStore<ReadModelEnvelope> = new DataStore(readModelsDatabase)
+  constructor() {
+    this.readModels.loadDatabase()
+  }
+
+  public async query(query: object): Promise<Array<ReadModelEnvelope>> {
+    const queryPromise = new Promise((resolve, reject) =>
+      this.readModels.find(query).exec((err, docs) => {
+        if (err) reject(err)
+        else resolve(docs)
+      })
+    )
+
+    return (await queryPromise) as Array<ReadModelEnvelope>
```

Comment:
I think that the await and the async modifiers are optional in this case. I wonder if this could work:

```suggestion
    return queryPromise as Promise<Array<ReadModelEnvelope>>
```

Reply:
Done in https://github.com/boostercloud/booster/pull/473/commits/3c8b3a4c2f22227b10e16c3689c8c1f4c03e051d